debian/control

```diff
@@ -39,8 +39,10 @@ Depends: eos-shell-content (= ${binary:Version}),
          ${misc:Depends},
          ${perl:Depends},
          debhelper,
+         libappstream-glib,
```

Comment:
There's actually no libappstream-glib package. What you want is gir1.2-appstreamglib-1.0.